references/indexes.md

```diff
@@ -129,11 +129,11 @@ Update an index.
 
 | Variable          | Description           |
 |-------------------|-----------------------|
-| **identifier** | the document <glossary word="attribute" /> of its unique identifier |
+| **primaryKey** | The <glossary word="primary key" /> of the documents  |
 
 The `uid` of an index cannot be changed. The document identifier `identifier` can be added if it does not already exist (to now if it has ben set, use [the get index route](/references/indexes.md#get-one-index)).
 
-[There are many ways in MeiliSearch to set the identifier](/guides/main_concepts/documents.md#identifier).
+[There are many ways in MeiliSearch to set the identifier](/guides/main_concepts/documents.md#primary-key).
```

Comment:
```suggestion
[There are many ways in MeiliSearch to set the primary key](/guides/main_concepts/documents.md#primary-key).
```

Reply:
This one slipped, thanks for the eagle eyes